hw2_epam_lab/src/main/java/hw2/Fibonachi.java

```diff
@@ -0,0 +1,49 @@
+package hw2;
+
+import java.util.ArrayList;
+
+public class Fibonachi {
+
+    public static String сalculate(int loopType, int n) {
+        ArrayList<Integer> fibanachi = new ArrayList<>(n);
```

Comment:
Don't use ArrayList to define new list

List<Integer> fibonacci= new ArrayList<>(n);